docs/guides/test-capability.md

```diff
@@ -0,0 +1,638 @@
+
+# Test Capability
+
+## Setup
+
+This guide assumes you have a project set up with Superface installed. If you need to set up a new project, please reference the [Setup Guide](./setup-the-environment.md).
+
+### Prerequisites
+
+- Existing profile
+- Existing [provider definition](./add-new-provider.md)
+- Existing [map between profile & the provider](./map-capability-to-provider.md)
+- [Superface CLI](/reference/cli) installed
+- [Superface OneSDK](/reference/one-sdk-js) installed
+
+## Configure existing profile and map {#configure-profile-map}
+
+To run and test your capabilities, you have to have configured profile and map in `super.json`.
+
+### Configuring profile {#configure-profile}
+
+To configure local profile, within your `super.json`, you have to add object representation of profile inside `profiles`, it's key have to be profile id matching id inside referenced file and value is object containing file path and providers.
+
+```json title="superface/super.json" {3-7}
+{
+  "profiles": {
+    "<profile-id>": {
+      "file": "./profiles/my-profile.supr",
+      "providers": {
+        // ...
+      }
+    }
+  },
+  "providers": {
+    // ...
+  }
+}
+```
+
+### Configuring map {#configure-profile}
+
+To configure local map, you have to add object representation of map with file path similar to profile specification.
+
+```json title="superface/super.json" {6-8}
+{
+  "profiles": {
+    "<profile-id>": {
+      "file": "./profiles/my-profile.supr",
+      "providers": {
+        "<provider-id>": {
+          "file": "./maps/my-map.suma"
+        }
+      }
+    }
+  },
+  "providers": {
+    // ...
+  }
+}
+```
+
+:::info
+
+To know more about profiles, maps and their configurations in `super.json`, check out our documentation about **[Comlink](/comlink)**
+
+:::
```

Comment:
Comlink reference doesn't talk about super.json, I think this is not covered by docs yet?

Reply:
I think Lukáš is working on it, should I hide it for now?

Reply:
Agreed :+1: